### DISCUSSION TITLE
Question about SuperSlicer

### DISCUSSION THREAD

**Discussion by MihkelBaranov** [bassamanator/Sovol-SV06-firmware#10] [Q&A] [answered]
Hi, 
Thanks for all the work you've done so far.
I'm quite new to SuperSlicer, can someone do a quick walkthrough of how to use/import the super slicer config files?

**Comment by bassamanator** [comment, marked as answer]
Please see a partial guide [here](https://github.com/bassamanator/Sovol-SV06-firmware/discussions/11).
